design-system/src/_includes/pattern-library/foundations/foundations-links/links.html

```diff
@@ -1,2 +1,17 @@
 
-<p><a href="#">Start making your will</a></p>
+<!-- Text link -->
+<p><a class="coop-t-link" href="#">Start making your will</a></p>
+
+<h3>Links with icons</h3>
+
+<!-- Text link with arrow SVG (pink) -->
+<p><a class="coop-t-link coop-t-link--arrow" href="#">More about offers<svg class="coop-t-link__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 20" aria-hidden="true" focusable="false"><path class="coop-u-brand-membership-pink-bright-fill" d="M13.89 0L11.4 2.46l5.87 5.8H0v3.48h17.28l-5.88 5.8L13.89 20 24 10z" stroke="none"/></svg></a></p>
+
+<!-- Text link with arrow SVG (orange) -->
+<p><a class="coop-t-link coop-t-link--arrow" href="#">More about offers<svg class="coop-t-link__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 20" aria-hidden="true" focusable="false"><path class="coop-u-brand-membership-orange-bright-fill" d="M13.89 0L11.4 2.46l5.87 5.8H0v3.48h17.28l-5.88 5.8L13.89 20 24 10z" stroke="none"/></svg></a></p>
```

Comment:
`coop-u-brand-membership-orange-bright-fill` 😢

Reply:
I know 😭 

I could have made them all blue with `coop-u-brand-fill`—that's a nice short one??